components/abtesting/optimizelyXExperiment/src/optimizely-x.js

```diff
@@ -19,7 +19,8 @@ const waitUntil = (truthyFn, callback, delay = 100, interval = 100) => {
   }, interval)
 }
 
-const getOptmizely = () => window && window.optimizely
+const getOptmizely = () =>
+  window && window.optimizely && window.optimizely.get && window.optimizely
```

Comment:
is intentional to evalueate window.optimizely twice?

Reply:
Seems to be done to force return on the left to right conditional evaluation.

Reply:
I need to return window.optimizely only if window.optimizely.get exists.